### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by ImpossibleAd436** [Bing-su/adetailer#519]
### Is your feature request related to a problem? Please describe.

The problem: After detailer prompts can be seperated using [SEP] but the order in which these are applied currently depends on the size of the bounding box, which may or may not result in the desired order

### Describe the solution you'd like

According to the Adetailer settings, bounding boxes can be "sorted" by position, left to right. I'm not 100% clear on what this does, I had expected it to do what I am suggesting here, but that doesn't seem to be the case.
If it is the case that Adetailer can sort bounding boxes from left to right, then in some cases it may be known which characters will be in which order from left to right, so an option to change the order of seperate prompts being applied (currently sorted by size) to being sorted by left to right would be useful. Another option which would be potentially useful would be to sort by confidence, and since this is a ratio which Adetailer will also know, perhaps that option could be added to.

In short, I would like to be able to better control the order of A {SEP] B [SEP] C, using size, left to right, and confidence, not only size as it is currently.
